apps/console/src/features/identity-providers/pages/identity-provider-edit.tsx

```diff
@@ -565,6 +565,7 @@ const IdentityProviderEditPage: FunctionComponent<IDPEditPagePropsInterface> = (
 
     return (
         <PageLayout
+            pageTitle={ "Edit Connection" }
```

Comment:
Let's remove the unnecessary curly brackets.

Reply:
Fixed with 509f01e